contracts/Bridge.sol

```diff
@@ -30,12 +30,11 @@ contract Bridge is Adminable {
     bytes32[] children; // unordered list of children below this node
   }
 
-  bytes32 public constant GENESIS = 0x4920616d207665727920616e6772792c20627574206974207761732066756e21;
+  bytes32 constant GENESIS = 0x4920616d207665727920616e6772792c20627574206974207761732066756e21;
 
-  uint256 public genesisBlockNumber; 
   bytes32 public tipHash; // hash of first period that has extended chain to some height
-  uint256 public parentBlockInterval; // how often epochs can be submitted max
-  uint64 public lastParentBlock; // last ethereum block when epoch was submitted
+  uint256 parentBlockInterval; // how often epochs can be submitted max
```

Comment:
I suggest to keep it public and drop the getter

Reply:
i read somewhere that getters should be preferred over the public modifier. can't find the source, but consider it best practice to avoid public modifier at all.

Reply:
Will break node I think.

Reply:
don't see a use of this in node